### DISCUSSION TITLE
Maps frequency update

### DISCUSSION THREAD

**Discussion by severinmenard** [organicmaps/organicmaps#7516] [Q&A] [answered]
I cannot see any information in the app about when the maps have been created. Is it smth you plan to add?
Is there a regular update frequency for the maps (like every month around day 1, as OsmAnd)? If so, it would be good to mention it in [Organic Maps' OSM wikipage](https://wiki.openstreetmap.org/wiki/Organic_Maps) or in the GitHub frontpage

**Comment by Misalf-git** [comment, marked as answer]
> I cannot see any information in the app about when the maps have been created.

See the "About" screen, reachable by tabbing the "Organic Maps"-logo (bottom left on map screen) or "Settings" (bottom right) -> "Help" (bottom).

> Is there a regular update frequency for the maps ...

Maps data is bound to app releases, which happen about once a month.

**Comment by severinmenard** [comment]
Thanks for the answer. 
I missed the mention and was expecting to see the map date in the map downloads.
About map data being bound to app release, this is quite unusual, I would not have imagined that first need to download the latest version of the app before being proposed more recent maps. And actually I have been checking for such map update for a few weeks, is there any (technical or otherwise) reason for this? Or is it something that might evolve in the future?

**Reply by Misalf-git** [reply]
> was expecting to see the map date in the map downloads.

Sounds reasonable to me. Also this was requested at least once.
- #525


> About map data 

I don't actually have a real clue, but the maps files contain OSM data in Organic Map's own format where features/rules/logic get added and/or changed frequently, plus wikipedia articles and elevation data and probably more.

I _think_ version-agnostic maps data is on the devs wish list, no idea about todo list.

**Reply by biodranik** [reply]
Yes, it is technically complex. Yes, it is planned. It may take a lot of working hours to implement and test. As map data format is bound to the app changes, the app should be updated anyway sometimes to proceed with newer, better, different maps format.

**Reply by pastk** [reply]
- #835
